### DISCUSSION TITLE
Star wars api returns wrong id

### DISCUSSION THREAD

**Discussion by ZakkProjects** [vsb-vaj/.github#10] [Q&A] [answered]
Is the behavior of the Star Wars API correct ?

When requesting for example https://swapi.dev/api/films/1/?format=json, movie with id 4 is returned instead of 1.

**Comment by petrvecera** [comment, marked as answer]
Hi 

It's recommended to use this API `https://swapi.profiq.com/api/films/1` which is different from swapi.dev, the original API has various bugs. 

On this link you will get `ID` and `EPISODE_ID` which are 2 different things.
